src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -160,6 +163,18 @@ export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>
   // APR change
   const newDebt = { nativeAmount: actionNativeCryptoAmount, tokenId: selectedAsset.tokenId, apr: newDebtApr }
 
+  // LTV exceeded checks
+  const pendingDebts = isSceneTypeDebts ? [...debts, pendingDebtOrCollateral] : debts
+  const pendingDebtsFiatValue = useTotalFiatAmount(borrowEngineWallet, pendingDebts)
+  const pendingCollaterals = isSceneTypeDebts ? collaterals : [...collaterals, pendingDebtOrCollateral]
+  const pendingCollateralsFiatValue = useTotalFiatAmount(borrowEngineWallet, pendingCollaterals)
+
+  // TODO: When new asset support is added, we need to implement calculation of aggregated liquidation thresholds
+  const hardLtvRatio = '0.74'
+  const isLtvExceeded =
```

Comment:
`isLtvExceeded` should eb included in the boolean for `sliderDisabled` on `FormScene`?